<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="01250LT2368" name="testCase.TestCaseForLogin" tests="9" failures="1" timestamp="25 Nov 2015 18:06:50 GMT" time="21.694" errors="0">
  <testcase name="verifyInvalidID" time="5.652" classname="testCase.TestCaseForLogin"/>
  <testcase name="verifyTitle" time="0.206" classname="testCase.TestCaseForLogin"/>
  <testcase name="logout" time="4.644" classname="testCase.TestCaseForLogin"/>
  <testcase name="navigateToAccountPage" time="0.017" classname="testCase.TestCaseForLogin">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at pageObject.NavigateToOtherPage.navigateToAccountCreatePage(NavigateToOtherPage.java:35)
	at testCase.TestCaseForLogin.navigateToAccountPage(TestCaseForLogin.java:252)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:207)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:178)
]]>
    </failure>
  </testcase> <!-- navigateToAccountPage -->
  <testcase name="logIn" time="6.399" classname="testCase.TestCaseForLogin"/>
  <testcase name="verifyRemebarMeCheckBox" time="1.400" classname="testCase.TestCaseForLogin"/>
  <testcase name="verifyInvalidPassword" time="1.245" classname="testCase.TestCaseForLogin"/>
  <testcase name="verifyInvalidIDBlank" time="1.319" classname="testCase.TestCaseForLogin"/>
  <testcase name="validLogin" time="0.812" classname="testCase.TestCaseForLogin"/>
</testsuite> <!-- testCase.TestCaseForLogin -->
